### PR TITLE
remove font awesome and add fa-pencil inline svg

### LINKED ISSUE #150: font awesome is loaded just to show one pencil icon
It's 7kb of css and 76kbs of fonts. I think inline SVG should be better, unless there's a plan to have more icons in the future.

### PR DESCRIPTION
**Pull request by MartinMalinda**
fixes #150

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,6 @@
     "ember-data": "^2.11.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.5",
-    "ember-font-awesome": "2.2.0",
```
Comment (sivakumar-kailasam) [inline package.json]: @MartinMalinda can you remove the dependency from the yarn.lock too? There's a corresponding bower dependency as well.

**Comment by sivakumar-kailasam** [comment]
There are some fontello related files in styles folder that can be removed with this PR since they aren't being used.

**Comment by MartinMalinda** [comment]
I have also removed select2 styles while I was at it. Maybe the swiftype css could be removed as well?

**Comment by toddjordan** [comment]
Also unsure why we are pinned to a github commit of select2 as opposed to semantic versions.  That can be part of a separate pr updating dependencies though.  Pulling this down now to check it out...

**Comment by sivakumar-kailasam** [comment]
Speaking of select, api-docs uses emberx-select & guides uses power select (think so). Should we move to power select as well?

File: app/templates/project-version/class.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <article class="chapter">
-  <a class="heading__link__edit" href="{{github-link model.project.id model.projectVersion.version model.file model.line}}" target="_blank" rel="noopener">{{fa-icon "pencil"}}</a>
+  <a class="heading__link__edit" href="{{github-link model.project.id model.projectVersion.version model.file model.line}}" target="_blank" rel="noopener">{{partial "svg/fa-pencil"}}</a>
```
Comment (MartinMalinda) [inline app/templates/project-version/class.hbs]: I can add a simple one via CSS pseudo element, reading from some data-title attr.

Comment (toddjordan) [inline app/templates/project-version/class.hbs]: i'll go ahead and merge.  It can be done in a separate pr

**Pull request merged**